history-of-quakers/modernized/10-chapter-8.adoc

```diff
@@ -1408,6 +1505,10 @@ and that they had been banished out of the country.
 Now to enter upon this bloody business, the following act was made at a general court,
 held at Boston, the 20th of October, in this year, 1658.
 
+[.embedded-content-document.legal]
```

Comment:
I wasn't sure if this act should be embedded or a block quote.

Reply:
i could see it going either way, but I think i would have picked embedded legal too. 👍